src/fvtt/page-script.js

```diff
@@ -25,9 +25,25 @@ class FVTTDisplayer {
 
     _postChatMessage(message, character, whisper, play_sound = false, attack_rolls, damage_rolls) {
         const MESSAGE_TYPES = CONST.CHAT_MESSAGE_TYPES || CHAT_MESSAGE_TYPES;
-        const pool = new DicePool([...attack_rolls, ...damage_rolls.map(d => d[1])].map(r => r._roll));
+        let pool;
```

Comment:
Dont' know why it's :
```
let pool;
pool = new DicePool ...
```
instead of `const pool = new DicePool ...`

Reply:
Old holdover from a broken way to handle DnDBeyond Digital Dice Off vs. On